### DISCUSSION TITLE
Stewardship

### DISCUSSION THREAD

**Discussion by tarilabs** [dmn-tck/tck#501] [General]
After 5+ years, 100+ commits, 4M+ LoM revised, then 10+ editions/milestones publications of the TCK as both individual contributor _and_ stewardship of this project, for personal reasons I will be spending time contributing to additional open source projects, so I won’t be able to continue on with the stewardship of the TCK.

**Comment by falko** [comment]
Hi Matteo,

thank you for being upfront about it. Do you have any suggestions on who of the active members would be a good successor? Out of curiosity, which projects are you going to work on? Anything DMN-related?

Cheers,
Falko

**Reply by tarilabs** [reply]
If there is a non-Vendor candidate who participates in the OMG's RTF group, it would be ideal.

As an individual contributor here, I play _also_ a mediator with also the OMG's RTF, as I am working in both places anyway already. Same can be said of Octavian for instance; the work we do here, sometimes is reflected on the RTF and viceversa.

I'm planning to continue on that, on the existing premise that the TCK works aligns with the OMG's RTF; that's why I suggest someone who can keep bridges first-person too in OMG's would be ideal, imho.

**Comment by StrayAlien** [comment]
Hi @tarilabs - sorry to hear that.  You've been doing a great job.  Thank you.

Are you disappearing from the TCK completely or shall we still see you here at the TCK - participating, but not stewarding?

**Reply by tarilabs** [reply]
Thank you Greg, appreciated.

In my message, I was referring to the stewardship intentionally. Same per other comment.

**Comment by baldimir** [comment]
Hi everyone, if you would like, I can try to start doing the stewardship instead of Matteo. 

To introduce myself - I just joined IBM from Red Hat. In Red Hat, I was colleague with Matteo, working on the DMN and Drools from the quality engineering side, at the beginning of the DMN (I think I worked on 1.2 last if I remember it correctly). Then I moved to a manager role, but I missed coding so I accepted to join IBM as a software engineer working on Drools and DMN. There are like 2,5 years that I was working as a manager instead of coding, so there are definitely things that I need to catch up, so please be patient with me :) 

If you are ok with me taking the organizational things from Matteo, please let me know. Whoever is at the end stewarding this, I am looking forward to collaborate with all of you on the TCK suite.

**Comment by StrayAlien** [comment]
Tibor, very nice to meet you, and it would be an amazing thing if you did.
Most welcome indeed.

I'll be happy to help in whatever way I can, just let me know.

Many thanks,

Greg.

**Reply by baldimir** [reply]
Thanks Greg, what do you recommend as first steps for me please? I sent a request to join the DMN Google group as I see you are organizing the meetings etc. there. Do you think there is something else? If the group didn't have a meeting for some time, I can organize something, gathering the actual problems to discuss from the issues and PR list.

**Comment by StrayAlien** [comment]
Hi Tibor, it was actually Matteo that was organising meetings.  We do have
outstanding PRs, some on hold for reasons of clarification and so on, and
others that might be able to be merged.

So far, my main role has been submitting tests, but I do not have rights to
merge etc. That's okay. Often my tests relate to ambiguities or holes and
often reflect my interpretation and they require some discussion or
clarification. So me merging might not be a good idea!

Maybe it might just be good to get together to re-kick off stuff and see
where we are and where everybody's heads are at and how we go into tests
for 1.4 etc.

As usual, time zones are not easy. I'm in Melbourne, some others in
UK/Europe. Others in Canada /US zone. I'm happy to be flexible but 3am is a
stretch. 😁

Greg.

**Reply by baldimir** [reply]
Ok, I will try to organize something.

**Comment by StrayAlien** [comment]
Thank you!
